python/vyos/vyconf_session.py

```diff
@@ -95,6 +127,21 @@ def get_config(self):
             raise VyconfSessionError(self.output(out))
         return out.output
 
+    def show_sessions(
+        self, exclude_self: bool = False, exclude_other: bool = False
+    ) -> list | dict:
+        out = vyconf_client.send_request(
+            'show_sessions',
+            token=self.__token,
+            exclude_self=exclude_self,
+            exclude_other=exclude_other,
+        )
+
+        lst = json.loads(out.output)
+        if len(lst) == 1:
+            return lst[0]
```

Comment:
Maybe it would be better to return list always?.. I think it can be quite unexpected, and it will be hard to use in scripts if there will be such need.

Reply:
Yes, it's odd, but for certain functions designed for internal use, it can make some sense; cf.
https://github.com/vyos/vyos-1x/blob/current/python/vyos/utils/config.py#L27-L45